UPGRADE-7.4.md

```diff
@@ -163,6 +163,7 @@ Security
  * Deprecate `LazyFirewallContext::__invoke()`
  * Deprecate `PersistentTokenInterface::getClass()` and `RememberMeDetails::getUserFqcn()`, the user FQCN will be removed from the remember-me cookie in 8.0
  * Add argument `$accessDecision` to `AccessDecisionStrategyInterface::decide()`;
+ * Deprecated `UserAuthorizationCheckerInterface`, implement `UserGuestAuthorizationCheckerInterface` instead
```

Comment:
for 8.1 now

Reply:
Rebased and added in 9f3c4305df31a6273ee92bcbcd185bdf141da0f4